pkg/virt-operator/resource/apply/delete.go

```diff
@@ -463,11 +463,63 @@ func DeleteAll(kv *v1.KubeVirt,
 			return nil
 		}
 	}
+	kubeVirtLabelPrefix := "kubevirt.io/"
```

Comment:
nit: this string isn't necessary, you can use directly the string in the function since it isn't used in any other places